src/Moq/Protected/IProtectedAsMock.cs

```diff
@@ -96,6 +131,17 @@ public interface IProtectedAsMock<T, TAnalog> : IFluentInterface
 		/// <exception cref="MockException">The specified invocation did not occur (or did not occur the specified number of times).</exception>
 		void Verify<TResult>(Expression<Func<TAnalog, TResult>> expression, Times? times = null, string failMessage = null);
 
+		/// <summary>
+		///   Verifies that a property was set on the mock, specifying a failure message.
```

Comment:
```suggestion
		///   Verifies that a property was set on the mock.
```

I suspect you copied this summary from a `VerifySet` method overload where `failMessage` isn't optional. However, `failMessage` is optional here, so the summary is somewhat misleading and probably draws attention to `failMessage` more than necessary.